scripts/populate_mock_db.py

```
@@ -0,0 +1,63 @@
+        crew_list.append(
+            {
+                "name": f"Crew Member {i}",
+                "role": "FFC" if i % 5 == 0 else "FFD",
+                "contract_hours": "42.0 hrs",
+                "skills": "TTR LGV BA",
+            }
+        )
+
+    insert_crew_details(crew_list, db_conn=conn)
+
+    now = datetime.now()
```

Ruff (0.15.0)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)